<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 1px solid white;
border-radius: 10px;
background: white;

background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgba(72, 145, 217, 255), stop: 1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>711</width>
      <height>461</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>CRUDSRP</string>
     </attribute>
     <widget class="QLineEdit" name="recherche">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>30</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="tableau">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>71</y>
        <width>651</width>
        <height>331</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="stat">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Statistique </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Pdf</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ajout">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supp">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Arduino">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Arduino</string>
      </property>
     </widget>
     <widget class="QLabel" name="labela">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>410</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>mailing</string>
     </attribute>
     <widget class="QPushButton" name="sendBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>651</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="subject">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>44</y>
        <width>651</width>
        <height>16</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>subject</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseBtn">
      <property name="geometry">
       <rect>
        <x>587</x>
        <y>79</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse ...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="file">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>571</width>
        <height>23</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>attachement here</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="msg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>116</y>
        <width>651</width>
        <height>291</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>type your message here ...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rcpt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>recipient</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mail_pass">
      <property name="geometry">
       <rect>
        <x>522</x>
        <y>10</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>your mail password</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Localisation</string>
     </attribute>
     <widget class="WebAxWidget" name="WebBrowser">
      <property name="control" stdset="0">
       <string>{8856F961-340A-11D0-A96B-00C04FD705A2}</string>
      </property>
      <property name="focusPolicy" stdset="0">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="geometry" stdset="0">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>691</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionadd">
   <property name="text">
    <string>add</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actiondelete">
   <property name="text">
    <string>delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionplay_video">
   <property name="text">
    <string>play video</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionstop_video">
   <property name="text">
    <string>stop video</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
  <customwidget>
   <class>WebAxWidget</class>
   <extends>QAxWidget</extends>
   <header>webaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
